editor/animation_track_editor.cpp

```diff
@@ -7653,6 +7692,14 @@ AnimationTrackEditor::AnimationTrackEditor() {
 
 	bottom_hf->add_child(selected_filter);
 
+	alphabetic_sorting = memnew(Button);
+	alphabetic_sorting->set_flat(true);
+	alphabetic_sorting->connect(SceneStringName(pressed), callable_mp(this, &AnimationTrackEditor::_update_tracks));
+	alphabetic_sorting->set_toggle_mode(true);
+	alphabetic_sorting->set_tooltip_text(TTR("Sort tracks/groups alphabetically.\nIf disabled, tracks are shown in the order they are added and can be reordered using drag-and-drop."));
```

Comment:
This doesn't appear to be true after testing it. I was convinced it was possible to drag-and-drop to reorder though, so I remember it wrong.

```suggestion
	alphabetic_sorting->set_tooltip_text(TTR("Sort tracks/groups alphabetically.\nIf disabled, tracks are shown in the order they are added."));
```

Reply:
I don't think this is right. You can reorder tracks, just only within the group when in grouped mode, which makes sense. I made a change so you can do this when the groups are sorted alphabetically as well, but of course it makes no sense to allow in tracks only view.

Reply:
I see… What I was trying to achieve was to reorder groups around.